### PR TITLE
Improve `plot_force` and fix `spec_updater`

### PR DESCRIPTION
**Pull request by sophietheis**

### CODE REVIEW

**Comment by glyg** [comment]
Just lacks a small test :)

File: tyssue/utils/utils.py [hunk 1]
```diff
@@ -58,9 +58,12 @@ def spec_updater(specs, new):
     specs: specification that will be modified
     new: dictionary of new specification
     """
-    for key, spec in specs.items():
-        if new.get(key) is not None:
-            spec.update(new[key])
+    for key, spec in new.items():
```
Comment (glyg) [inline tyssue/utils/utils.py:61]: just `for key in specs` as the `spec` variable is not used

File: tests/draw/test_plt.py [hunk 2]
```diff
@@ -15,7 +15,7 @@
 from tyssue import Sheet, config
 from tyssue.draw.plt_draw import quick_edge_draw, sheet_view
 from tyssue.draw.plt_draw import _face_color_from_sequence
-from tyssue.draw.plt_draw import create_gif
+from tyssue.draw.plt_draw import create_gif, plot_forceœ
```
Comment (glyg) [inline tests/draw/test_plt.py]: oups there's a typo

Comment (sophietheis) [reply tests/draw/test_plt.py]: I just fix it, the magic of key combination, I really don't know how to do this letter...

File: tests/draw/test_plt.py [hunk 3]
```diff
@@ -15,7 +15,7 @@
 from tyssue import Sheet, config
 from tyssue.draw.plt_draw import quick_edge_draw, sheet_view
 from tyssue.draw.plt_draw import _face_color_from_sequence
-from tyssue.draw.plt_draw import create_gif
+from tyssue.draw.plt_draw import create_gif, plot_force
```
Comment (glyg) [inline tests/draw/test_plt.py]: it's `plot_forces` :laughing:

Comment (sophietheis) [reply tests/draw/test_plt.py]: It's difficult to write test this morning..., I also forgot to put a test, in the test function... --'

**Comment by glyg** [comment]
You can add sheet.update_spec(model.specs) to have the correct columns (and a test that passes)

**Comment by glyg** [comment]
Fridays are hard

**Pull request merged**